openstack/messaging/v2/messages/testing/requests_test.go

```diff
@@ -0,0 +1,41 @@
+package testing
+
+import (
+	"testing"
+
+	"github.com/gophercloud/gophercloud/openstack/messaging/v2/messages"
+	th "github.com/gophercloud/gophercloud/testhelper"
+	fake "github.com/gophercloud/gophercloud/testhelper/client"
+)
+
+func TestCreate(t *testing.T) {
+	th.SetupHTTP()
+	defer th.TeardownHTTP()
+	HandleCreateSuccessfully(t)
+
+	createOpts := messages.CreateOpts{
+		QueueName: QueueName,
+		Messages: []messages.Messages{
+			{
+				TTL:   300,
+				Delay: 20,
+				Body: map[string]interface{}{
+					"event":     "BackupStarted",
+					"bakcup_id": "c378813c-3f0b-11e2-ad92-7823d2b0f3ce",
+				},
+			},
+			{
+				Body: map[string]interface{}{
+					"event":         "BackupProgress",
+					"current_bytes": "0",
+					"total_bytes":   "99614720",
+				},
+			},
+		},
+	}
+
+	blah := messages.Create(fake.ServiceClient(), createOpts)
```

Comment:
`s/blah/result/` :)

Or you could do:

```go
actual, err := messages.Create(...).Extract()
```

Reply:
I don't know how that got in there.... sorry about that.

Reply:
No problem at all 😄